.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.3"
+  - "3.6"
+  - "3.7"
 env:
   - DJANGO_VERSION=1.4
   - DJANGO_VERSION=1.5
   - DJANGO_VERSION=1.10
   - DJANGO_VERSION=2.0
 install:
   - pip install Django==$DJANGO_VERSION
+  - pip install requests
+  - pip install Pillow
```

Comment:
Funny enough I'm noticing we are not requiring the dependencies: https://github.com/vinyll/django-imagefit/blob/master/setup.py#L17 🤔 

Can you add them 🙏 ?